apis/extension/cpu_normalization.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package extension
+
+import (
+	"fmt"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// LabelCPUNormalizationRatio denotes the cpu-normalization ratio of the node CPU.
+	LabelCPUNormalizationRatio = NodeDomainPrefix + "/cpu-normalization-ratio"
```

Comment:
better to be an annotation since it won't be used for select and can be extended as a struct with other overcommit fields.

Reply:
done